src/graphql/mutations.js

```diff
@@ -221,7 +221,24 @@ export const createVirtualCardsMutationQuery = gql`
 
 export const addCreateOrderMutation = graphql(createOrderQuery, {
   props: ({ mutate }) => ({
-    createOrder: order => mutate({ variables: { order } }),
+    createOrder: async order => {
+      return await mutate({
+        variables: { order },
+        update: (cache, { data: { createOrder } }) => {
+          const data = cache.readQuery({
+            query: getMutatedCollective,
+            variables: { slug: createOrder.collective.slug },
+          });
+          data.Collective.orders.push({
```

Comment:
Why not pushing the entier `createOrder` object?

Reply:
I'm not exactly sure, at first glance of the ```createOrder```  object didn't seem to return information with the same shape as the orders in the collective in cache so I didn't think much of it and just added the specific fields that were relevant. 

However, looking at the documentation, you are right, I should probably just push the entire object as is. I will update the pull request accordingly.